.travis.yml

```diff
@@ -35,7 +36,8 @@ before_install:
     fi
 
 before_script:
-- curl -L --create-dirs -o $HOME/.sbt/launchers/1.2.6/sbt-launch.jar http://dl.bintray.com/sbt/maven-releases/org/scala-sbt/sbt-launch/1.2.6/sbt-launch.jar
+- .travis/download-sbt
+- . .travis/download-graalvm
```

Comment:
Why two dots?

Reply:
Thanks for the explanation 🤗 too long bash abstinence 😁
So this is needed in order to make the export work, right?

Reply:
Indeed. There could be other ways but this is the neatest.

```bash
$ bash <(echo export x=y)
$ echo $x

$ source <(echo export x=y)
$ echo $x
y
$
```